src/movies.js

```diff
@@ -1,17 +1,142 @@
 // Iteration 1: All directors? - Get the array of all directors.
 
+function getAllDirectors(movies) {
+    const mydirectors = movies.map((x) => x.director);
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+return mydirectors.filter((x, index) => mydirectors.indexOf(x) === index);
```

Comment:
It will more clear I guess, if instead of (x) you put exactly what are you looking for, but anyways, good work!